libnmstate/nm/ovs.py

```diff
@@ -128,18 +130,10 @@ def get_port_by_slave(nmdev):
     return None
 
 
-def get_bridge_info(bridge_device, devices_info):
-    info = get_ovs_info(bridge_device, devices_info)
-    if info:
-        return {OB.CONFIG_SUBTREE: info}
-    else:
-        return {}
```

Comment:
What happened to this section?
I do not see it under `nm.ovs` now. Can you leave it here and then move it in a following commit? It will help make sure all is ok with this move.
If it complicates things, do not bother, just respond with what happened.

Reply:
The function `get_bridge_info` is not used by production code but test code only.

I will create separate commit to remove it.

Reply:
PR https://github.com/nmstate/nmstate/pull/1050 is created to remove the dead code.